index.html

```diff
@@ -1,25 +1,31 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <link rel="stylesheet" href="style.css">
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="style.css" />
         <title>Landing Page</title>
     </head>
 
     <body>
         <div class="header">
             <div class="top">
                 <h1>Header Logo</h1>
-                <a href="#"> header link one</a>
-                <a href="#"> header link two</a>
-                <a href="#"> header link three</a>
+                <nav class="nav">
```

Comment:
Another way to space the logo apart from the links is to wrap the links in an element (usually a good idea anyway) then add `justify-content: space-between` to `.top` to force the two children (`<h1 />` and `<nav />`) apart. 

Example: https://github.com/VamanaPed/LandingPage/pull/1/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aR39